http/src/routing/route.rs

```diff
@@ -458,6 +458,13 @@ pub enum Route {
         /// The maximum number of guilds to get.
         limit: Option<u64>,
     },
+    /// Route information to get an original interaction response message.
+    GetInteractionOriginal {
+        /// The ID of the owner application.
```

Comment:
I know that these look consistent with the other docs, but we try not to start doc sentences with `The`, otherwise, they would all start with `The`.
```suggestion
        /// ID of the owner application.
```